scheduling/dispatcher.rb

```diff
@@ -1,14 +1,17 @@
 # frozen_string_literal: true
 
 class Scheduling::Dispatcher
-  attr_reader :notifiers
+  attr_reader :notifiers, :shutdown
 
   def initialize
     @apoptosis_timeout = Strand::LEASE_EXPIRATION - 29
     @notifiers = []
+    @shutdown = false
+    Signal.trap("TERM") { @shutdown = true }
```

Comment:
I appreciate the attempt to get a test in on this one, but given how non-compositional signal handlers are, I might have put this one in `bin/respirate` and left it untested (as all `bin` program texts are). 

In doing so, I'd perhaps upgrade `shutdown` to a method that is monotonic (can only set @shutdown to true, though naming perhaps should be revised to not clash with the method).

Reply:
Moved to `bin/respirate`.